ocs_ci/framework/conf/ocs_version/ocs-4.99.yaml

```diff
@@ -0,0 +1,12 @@
+---
+DEPLOYMENT:
+  default_ocs_registry_image: "quay.io/rhceph-dev/ocs-registry:latest-4.99"
+  default_latest_tag: 'latest-4.99'
+  ocs_csv_channel: "stable-4.9"
+  default_ocp_version: '4.9'
```

Comment:
shouldn't we put default ocp-version to be latest i.e 4.11?

Reply:
Yes for now on OCP 4.11. 
And you are right mapping should be same for odf:ocp but since we don't have any 4.99 ocp version and this is for master development it will always take the latest OCP versions

Reply:
done